.travis.sh

```diff
@@ -12,7 +12,12 @@ set -e # Error build immediately if install script exits with non-zero
 # if this is a docker job, run in the container instead; but if not just run it here.
 if [ -n "$DOCKER_IMAGE" ]; then
     ./util/dockerize.sh ./.ci.sh
+elif [ "${TRAVIS_OS_NAME}" = "windows" ]; then
+    choco install wsl-ubuntu-1804
+    brun="/c/ProgramData/chocolatey/lib/wsl-ubuntu-1804/tools/unzipped/ubuntu1804.exe run"
+    $brun sudo apt update
+    $brun sudo apt install -y gcc git curl
+    $brun ./.ci.sh
```

Comment:
We need to demonstrate that this actually errors out if there are bugs. When I was working on #2718 I had at least one case where there were failing tests but CI passed. Someone (me or you) needs to demonstrate with screenshots:

- a failed pytest test
- a `exit 9` in an obscure place in .ci.sh
- a failed integration test

Reply:
There's some evidence that at least using the ubuntu binary errors properly on a test fail. https://travis-ci.org/github/neuropoly/spinalcordtoolbox/builds/688649596